daniellamarr/warren-web
# Add partner logos

#### What does this PR do?
- Currently, the partner's section is empty which is not meant to be. The aim of this task is to add partner logos on the partner section
#### Description of Task to be completed?
- Add partner logos
- Make partner section responsive
- Make partner section scrollable
#### How should this be manually tested?
- Pull this branch `factor-in-partner-logos`
- Navigate to the `index.html` file
- Locate the `Partners` link on the navbar and click on it
- You should now be able to view the implementation
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-04-30 at 2 30 27 PM" src="https://user-images.githubusercontent.com/42177767/56965156-8851d500-6b54-11e9-9e75-9b083addc3fc.png">
